test/Feature/WriteCov.c

```diff
@@ -1,21 +1,18 @@
 // RUN: %llvmgcc %s -emit-llvm -g -c -o %t2.bc
 // RUN: rm -rf %t.klee-out
 // RUN: %klee --output-dir=%t.klee-out --exit-on-error --write-cov %t2.bc
-// RUN: grep -c WriteCov.c:16  %t.klee-out/test000001.cov %t.klee-out/test000002.cov  >%t3.txt
-// RUN: grep -c WriteCov.c:18  %t.klee-out/test000001.cov %t.klee-out/test000002.cov >>%t3.txt
-// RUN: grep %t.klee-out/test000001.cov:0 %t3.txt
```

Comment:
Did you mean to remove the greps in `.cov` files too?

Reply:
Yes, because they are replaced by the `FileCheck`

Reply:
I guess that's good enough, but I don't think it's checking the same thing, is it?  If both `.cov` files had `WriteCov.c:16` and ` WriteCov.c:18`, the test would now pass, correct?

Reply:
@ccadar Agreed, I removed the changes for `WriteCov.c`